docs/data/data-grid/layout/layout.md

```diff
@@ -8,6 +8,20 @@ Instead, it takes up the space given by its parent.
 The Data Grid will raise an error in the console if its container has no intrinsic dimensions.
 :::
 
+## Flex parent container
+
+The Data Grid can be placed inside a flex container with `flex-direction: column`.
+Without setting the minimum and maximum height, the Data Grid will take as much space as it needs to display all rows.
+This is effectively equivalent to setting `autoHeight={true}`, but with the added benefit of being able to set the minimum and maximum height of the parent container.
```

Comment:
This line could be placed in a callout below the demo with a link to the **Auto height** section of the page.

Reply:
Done in https://deploy-preview-14614--material-ui-x.netlify.app/x/react-data-grid/layout/#flex-parent-container